app/public/static/js/gallery_carousel.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+const galleryCarousel = new bootstrap.Carousel(document.querySelector('#galleryCarousel'));
+document.querySelectorAll('.gallery-image').forEach(el => {
+    el.addEventListener('click', function(e) {
+        galleryCarousel.to(el.dataset.bsSlideTo);
+    });
+});
```

Comment:
tabs for indentation + double quotes + final new line + e.preventDefault to stop opening link

```suggestion
window.addEventListener("load" () => {
	const galleryCarousel = new bootstrap.Carousel(document.getElementById("galleryCarousel"));
	document.querySelectorAll(".gallery-image").forEach(el => {
		el.addEventListener("click", function(e) {
			galleryCarousel.to(el.dataset.bsSlideTo);
			e.preventDefault();
		});
	});
});

```